lib/infrastructure/core/dnd.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_dnd/flutter_dnd.dart';
+
+Future<void> toggleDND(BuildContext contect) async {
+  bool filter = await FlutterDnd.getCurrentInterruptionFilter() ==
+      FlutterDnd.INTERRUPTION_FILTER_PRIORITY;
+  if (await FlutterDnd.isNotificationPolicyAccessGranted ?? false) {
+    await FlutterDnd.setInterruptionFilter(
+      filter
+          ? FlutterDnd.INTERRUPTION_FILTER_ALL
+          : FlutterDnd.INTERRUPTION_FILTER_PRIORITY,
+    ); // Turn on DND - All notifications are suppressed.
```

Comment:
The comment describes the line, please move it before the line (above  await FlutterDnd.setInterruptionFilter)